source/simulator/core.cc

```diff
@@ -1043,6 +1046,7 @@ namespace aspect
             {
               coupling[x.velocities[d]][x.pressure] = DoFTools::always;
               coupling[x.pressure][x.velocities[d]] = DoFTools::always;
+              coupling[x.pressure][x.pressure] = DoFTools::always;
```

Comment:
@tjhei I looked into `signals.pre_compute_no_normal_flux_constraints` and that signal only takes a Triangulation as argument. Can I access the ConstraintMatrix `constraints` through this object?